packages/cli-kit/src/private/node/api.ts

```diff
@@ -120,6 +120,18 @@ async function makeVerboseRequest<T extends {headers: Headers; status: number}>(
   }
 }
 
+function errorsIncludeStatus429(error: ClientError): boolean {
+  if (error.response.status === 429) {
+    return true
+  }
+
+  // More so checking if type of error.response.errors is not GraphQLError[]
+  if (typeof error.response.errors === 'string') {
+    return false
+  }
+  return error.response.errors?.some((error) => error.extensions?.code === '429') ?? false
+}
```

Comment:
My TS isn't the best, ideally I want to do something like this but it didn't seem possible

```
if (typeof error.response.errros === 'GraphQL[]'){
return error.response.errors?.some((error) => error.extensions?.code === '429') ?? false
}
```

Reply:
I think the type checking here will fix [this issue](https://app.bugsnag.com/shopify/cli/errors/66c7d2ce7fce022d1c52629f). Thanks!

Reply:
I don't have access to this bugsnag, but I assume this is the `error.response.errors?.some is not a function` or something error I encountered as well?